empire.go

```diff
@@ -453,8 +457,56 @@ func (opts RunOpts) Validate(e *Empire) error {
 	return e.requireMessages(opts.Message)
 }
 
+func (e *Empire) confirm(ctx context.Context, user *User, action string, resource string, params map[string]string) error {
```

Comment:
This will fail closed if the third party is down (Duo), which I think is the right thing to do. If we failed open, an attacker could target the third party to gain access.